dlrover/python/elastic_agent/monitor/training.py

```diff
@@ -123,19 +123,9 @@ def report_resource_with_step(self):
         except Exception as e:
             logger.warning(e)
 
-    def send_heartbeat(self):
```

Comment:
@workingloong here

Reply:
You can add `fix #1413 ` in the description and the issue will be closed when the PR is merged.